.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: required
 dist: xenial
 language: go
 go:
-  - 1.17.x
+  - 1.18.x
```

Comment:
Why this bump?

Reply:
Our travis CI jobs were failing with below error:
```
make && make test
go build -tags ""  ./...
go vet ./...
GO111MODULE=off go get -u golang.org/x/lint/golint
# golang.org/x/tools/go/ast/astutil
../../../golang.org/x/tools/go/ast/astutil/enclosing.go:323:23: n.Type.TypeParams undefined (type *ast.FuncType has no field or method TypeParams)
```

This `golang.org/x/tools/internal/typeparams` package introduced in go 1.18.
https://pkg.go.dev/golang.org/x/tools/internal/typeparams

After updating go version for travis build, build passed.
https://app.travis-ci.com/github/libopenstorage/cloudops/builds/267823009